hedera-node/src/main/java/com/hedera/services/txns/token/TokenBurnTransitionLogic.java

```diff
@@ -92,10 +101,17 @@ public ResponseCodeEnum validate(TransactionBody txnBody) {
 			return INVALID_TOKEN_ID;
 		}
 
```

Comment:
Add Global Dynamic Property Check for exceeding the batch limit. See minting for reference